### PR TITLE
Feat/event server

### PR DESCRIPTION
**Pull request by lx1922**
### Issue
Closes <#29>

### What is changed / added
- bot scrapes events from first page of the website

### CODE REVIEW

File: src/api/bot.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+//     let eventDesc = document.querySelector('').innerText;
+
+//     return {
+//         title: eventTitle,
+//         date: eventDate,
+//         location: eventLoc,
+//         img: eventImg,
+//         description: eventDesc
+//     };
+// })
```
Comment (TheFatPanda97) [inline src/api/bot.js:29]: Let's remove these comments

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
Looks good, just needs to remove the extra comments

File: src/api/bot.js (hunk 1 above)
Comment (lx1922) [reply src/api/bot.js]: Done those were actually backup files which I had while working, but forgot to remove them! Removed them now!

**Review by TheFatPanda97 — COMMENTED** [review]
Let's merge master as well, the change log is not right

**Review by TheFatPanda97 — APPROVED** [review]
LGTM

**Pull request merged**